src/components/ec-inline-actions/ec-inline-actions.vue

```diff
@@ -74,61 +77,78 @@ export default {
       default: () => ({ offset: 10, placement: 'bottom-start' }),
     },
   },
+  methods: {
+    componentTag(item) {
+      if (item.href) {
+        return 'a';
+      }
+      return 'button';
+    },
+    getDownloadText(item) {
```

Comment:
I'm not sure here. Shouldn't the logic be:
1. if there is a href and download, return download
2. it there is a href and download is an empty string, return empty string <- MDN says download attribute doesn't have to have always a value
3. if there is a href and download is undefined or null, return null
4. if there is no href, return null

Reply:
I'm also adding the case that the item it's disabled, in that case, we will return null.

And the 3 and the 4 for me it's the same if you have the download undefined or null you will return null, doesn't matter the href.